plugins/traction_innkeeper/traction_innkeeper/v1_0/tenant/routes.py

```diff
@@ -109,6 +109,31 @@ async def setup_tenant_context(request: web.Request, handler):
     return await handler(request)
 
 
+def active_tenant_only(func):
+    @functools.wraps(func)
+    async def wrapper(request):
+        print("> innkeeper_only")
```

Comment:
Do you want this to be a print statement?

Reply:
Forgot to remove that on this branch. This code is now dropped